cmd/node/node.go

```diff
@@ -570,6 +588,32 @@ func (app *SpacemeshApp) checkTimeDrifts() {
 	}
 }
 
+func (app *SpacemeshApp) HareFactory(mdb *mesh.MeshDB, swarm service.Service, sgn hare.Signer, nodeID types.NodeId, syncer *sync.Syncer, msh *mesh.Mesh, hOracle hare.Rolacle, idStore *activation.IdentityStore, clock TickProvider, lg log.Log) HareService {
+	if app.Config.HARE.SuperHare {
```

Comment:
you could do this outside this func imo

Reply:
I've refactored this so this function decided which hare to take according to parameters

Reply:
imo super hare should go outside so it is noticed, but it's not super important ;)